### DISCUSSION TITLE
Can i send message to a specific user having their chat_id?

### DISCUSSION THREAD

**Discussion by pssolanki111** [python-telegram-bot/python-telegram-bot#2566] [Q&A] [answered]
Is it possible to send someone a message (that's me : D) if I have my chat_id and username? 

I already have the APscheduler code working on heroku and it works well and it can reply to messages received.

But instead of replying, I want to be able to send message proactively (of course after i start their convo with bot)?

what approach should i take?

Happy to provide more details if needed. 

Any suggestion is appreciated : )

**Comment by Bibo-Joshi** [comment, marked as answer]
Hi. Sending messages is as easy as calling `bot.send_message`. In PTB, the `bot` instance is usually available as `context/dispatcher/updater.bot`, but you can also instantiate it yourself, if needed. Please see [here](https://python-telegram-bot.readthedocs.io/en/stable/telegram.bot.html#telegram.Bot.send_message) and [here](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API).
